wrapper/src/libvmaf.h

```diff
@@ -99,23 +133,24 @@ class Result
     double get_score(const std::string &key);
     std::vector<std::string> get_keys();
     void setScoreAggregateMethod(ScoreAggregateMethod scoreAggregateMethod);
+    int get_num_frms();
+    void set_num_frms(int num_frms);
 private:
     std::map<std::string, StatVector> d;
     ScoreAggregateMethod score_aggregate_method;
+    int num_frms;
```

Comment:
unsigned

Reply:
Fixed. Need to fix other places where w and h are int, perhaps will rely on VmafPicture struct definition with unsigned int there and re-use that.